### DISCUSSION TITLE
IAC scan to capture Azure Application Gateway insecure TLS and cryptographic algorithms allowed

### DISCUSSION THREAD

**Discussion by tzurielweisberg** [aquasecurity/trivy#5145] [Ideas]
### Description

Hey,

A customer asked about addin IAC scan to capture Azure Application Gateway insecure TLS and cryptographic algorithms allowed. 

Can you please help us understand how can we take it into trivy support?

### Target

Filesystem

### Scanner

Misconfiguration

**Comment by simar7** [comment]
Is this a new policy you wish to add? If so, you can create a PR in the defsec repo. You can see examples here https://github.com/aquasecurity/defsec/tree/master/rules/cloud/policies/azure

**Reply by tzurielweisberg** [reply]
Thanks, moved to https://github.com/aquasecurity/trivy-policies/tree/main/checks/cloud/azure
